modules/repository-datastax/src/main/java/com/intuit/wasabi/repository/cassandra/impl/ExpBucket.java

```diff
@@ -0,0 +1,23 @@
+package com.intuit.wasabi.repository.cassandra.impl;
+
+
+import com.intuit.wasabi.experimentobjects.Experiment;
+import com.intuit.wasabi.experimentobjects.Bucket;
+
+
+class ExpBucket {
+    private Experiment.ID expID;
```

Comment:
@mooxter These should probably be `final` as well, as they shouldn't change, right?